agent/src/podman/workload.rs

```diff
@@ -18,94 +18,26 @@ use common::state_change_interface::StateChangeInterface;
 use common::state_change_interface::StateChangeSender;
 
 use hyper::http;
-use podman_api::models::{ContainerMount, PortMapping};
-use podman_api::opts::{ImageListFilter, ImageListOpts, PullOpts};
 use podman_api::Error;
```

Comment:
We should not have dependencies to podman_api here too.

Reply:
Fixed